src/Frontend-GNOME/Views/Chats/ChatView.cs

```diff
@@ -529,7 +529,7 @@ public virtual void Sync()
             DateTime start, stop;
             start = DateTime.UtcNow;
             // REMOTING CALL
-            SyncedMessages = _ChatModel.Messages;
+            SyncedMessages = _ChatModel.MessageBuffer;
```

Comment:
This one is complicated, it compiles?!? It needs special handling that I have added in a different branch, where I implemented the /sync with argument command.